app/modules/transcribe/annotations/marginalia.html

```diff
@@ -0,0 +1,3 @@
+<g class="marginalia-annotation">
+    <rect class="marginalia" draggable="data" ng-attr-x="{{ data.x }}" ng-attr-y="{{ data.y }}" ng-attr-height="{{ data.height }}" ng-attr-width="{{ data.width }}"></rect>
```

Comment:
I've seen this in a few places; separate lines after a certain number of attributes makes it easier to maintain.